### ISSUE TITLE
decreased plasma testosterone-to-androstenedione ratio

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5807]
**Preferred term label:**
Decreased plasma testosterone-to-androstenedione ratio

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/10599740/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as Decreased gonadotropin-stimulated testosterone-to-androstenedione ratio
